boards/amd/kv260_r5/board.cmake

```diff
@@ -1,3 +1,4 @@
 # Copyright (c) 2022 Linaro.
 #
 # SPDX-License-Identifier: Apache-2.0
+include(${ZEPHYR_BASE}/boards/common/xsdb.board.cmake)
```

Comment:
Shouldn't this get an `fsbl` argument? Or am I missing something?

Reply:
Is the `fsbl.elf` file generated from the build? Ideally the necessary files are passed by default to the runner, so users can simply call `west flash` without the need to add manual arguments.

Reply:
For example something like:

```cmake
board_runner_args(xsdb "--fsbl=${PROJECT_BINARY_DIR}/fsbl.elf") 
include(${ZEPHYR_BASE}/boards/common/xsdb.board.cmake)
```

Reply:
Thanks for the example, will update and document the usage in the commit message.

Reply:
> fsbl.elf is a bootloader binary built using AMD tools it won't be built using zephyr sources and this file is required inorder to initialize the hardware before loading the zephyr binaries.

Perhaps this should become a blob then?

Reply:
Not sure about blob but i think we should use file option in the RunnerConfig for this instead of introducing a new option fsbl and bitstream and pass that file from board_runner_args like you suggested above.  
file: Optional[str]             # binary file path (provided by the user), or None
let me know if this approach is fine for you or not, if you are okay will update the code accordingly.

Reply:
it's a manual action okay let me explore how west blobs fetch works

Reply:
The First Stage Boot Loader (fsbl.elf) is a design-specific (note: not only board-specific, but specific to the given PS/PL configuration used for a given design). It is is built outside of the Zephyr build environment and - in my experience - seldomly needs to be rebuilt.

Letting the user provide the path to the fsbl.elf in the runner perfectly aligns with how we use this downstream. I don't see how this would fit with the `west blobs` feature.

Reply:
Thanks @henrikbrixandersen for the inputs, @pdgendt existing/latest code aligns with the @henrikbrixandersen suggestion, if you are okay with the changes please approve the pull request

Regerds,
Kedar.